src/arithmetic/bigint.rs

```diff
@@ -1194,7 +1194,9 @@ fn limbs_mont_mul(r: &mut [Limb], a: &[Limb], m: &[Limb], n0: &N0) {
         target_arch = "aarch64",
         target_arch = "arm",
         target_arch = "x86_64",
-        target_arch = "x86"
+        target_arch = "x86",
+        target_arch = "mips",
+        target_arch = "mips64",
```

Comment:
Everywhere in this file, I would expect no conditional logic for MIPS because there is a default implementation for platforms for which we don't have assembly. If those default implementations are not working for MIPS, let's investigate together why not.